fredlemieux/eventos-rincon-nextjs
# Chore/add internalization

## Description
- Add next-intl and setup for es and en
- Add simple switch language button to the header.
- Will apply full translation throughout in another PR

## Screenshots
![image](https://github.com/user-attachments/assets/19532f62-dee0-4722-8eb3-a0c8b20f9cda)



## How to test
- Switch languages (URL root should change between es and en
- See console log test in NavItems this updates


## Checklist

- [ ] This does not have any known performance implications
- [ ] There are no risks involved
- [ ] This does not require any new/updated documentation
- [ ] Tests have been created to cover new/updated code

## Merging

- [ ] I promise to use Squash & Merge if this PR is going into `develop`
